listing-page/listingpage.css

```diff
@@ -76,10 +76,11 @@ section{
 /*cards*/
```

Comment:
changes: you have to use CSS variables for color.
![Screenshot 2022-02-20 at 2 24 23 PM](https://user-images.githubusercontent.com/79623546/154835382-f8466f2d-0bae-4209-b5fb-55e5f933e231.png)